### DISCUSSION TITLE
Lumber sorting and packing

### DISCUSSION THREAD

**Discussion by aaronhudon** [tui-cs/Terminal.Gui#2387] [Show and tell]
Thought I'd share the use of how Terminal.Gui has been to a project that has recently become viable.  The vast majority of these engines that are built  have a web interface to expose system operation.  Timelines were tight in this situation, and the ability to use Terminal.Gui to expose the operations and structures helped immensely.

![CMS 2023-03-01](https://user-images.githubusercontent.com/4406811/222178534-55fc828a-73f8-4456-9f67-0ee6d2b2970c.gif)

**Comment by tznind** [comment]
Awesome! That looks very cool.  What a lot of data!

Thanks for sharing.  Its always great to hear from people using the library :tada:

**Comment by BDisp** [comment]
That a great job, congratulations. The changes to allow to expose the `Terminal.Gui` capabilities for web interface was performed when I managed into this project https://github.com/Blazor-Console/HACC, which I haven't been time to continue for now.

**Comment by tig** [comment]
Thank you for sharing! This is amazing.
